src/vs/editor/contrib/stickyScroll/browser/stickyScroll.ts

```diff
@@ -315,11 +316,11 @@ class StickyScrollCodeLine {
 		innerLineNumberHTML.style.lineHeight = this._editor.getOption(EditorOption.lineHeight).toString() + 'px';
 		lineNumberHTMLNode.appendChild(innerLineNumberHTML);
 
-		root.onclick = e => {
+		dom.addDisposableListener(root, 'click', e => {
```

Comment:
we should record this disposable. We should use `this._register(dom.addDisposableListener(...));`